app/src/main/java/eu/kanade/tachiyomi/smartsearch/SmartSearchEngine.kt

```diff
@@ -127,13 +135,17 @@ class SmartSearchEngine(
      * @return a manga from the database.
      */
     suspend fun networkToLocalManga(sManga: SManga, sourceId: Long): Manga {
+        sManga.setTitleNormalized()
         var localManga = db.getManga(sManga.url, sourceId).executeAsBlocking()
         if (localManga == null) {
             val newManga = Manga.create(sManga.url, sManga.title, sourceId)
             newManga.copyFrom(sManga)
             val result = db.insertManga(newManga).executeAsBlocking()
             newManga.id = result.insertedId()
             localManga = newManga
+        } else if (localManga.title.isBlank() || localManga.title.contains("’")) {
+            localManga.title = sManga.title
+            db.insertManga(localManga).executeAsBlocking()
```

Comment:
i personally rather not fix this, leave this for extensions for now

Reply:
As you want, I'll revert it later. I decided to work on it after coming across [this bug](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/9322)  and I did it in the app to avoid doing it for each problematic extension and since U-2019 is quite difficult to do with regular keyboard on android but yeah it might be overkill to do it here just to avoid a few cases

Reply:
Fixing it for migration searching is fine with me, but let's not fix it in the db, or make changes in the ui for it